tfhe/examples/fhe_strings/server_key/mod.rs

```diff
@@ -313,29 +292,12 @@ impl ServerKey {
             &shifted,
         );
 
-        FheString::from_uint(result)
+        FheString::from_uint(result, false)
     }
 }
 
 pub trait FheStringIterator {
     fn next(&mut self, sk: &ServerKey) -> (FheString, BooleanBlock);
 }
 
-#[derive(Clone)]
-enum CharIter<'a> {
-    Iter(std::slice::Iter<'a, FheAsciiChar>),
-    Extended(
-        std::iter::Chain<std::slice::Iter<'a, FheAsciiChar>, std::iter::Once<&'a FheAsciiChar>>,
-    ),
-}
-
-impl<'a> Iterator for CharIter<'a> {
-    type Item = &'a FheAsciiChar;
-
-    fn next(&mut self) -> Option<Self::Item> {
-        match self {
-            CharIter::Iter(iter) => iter.next(),
-            CharIter::Extended(iter) => iter.next(),
-        }
-    }
-}
+type CharIter<'a> = Vec<&'a FheAsciiChar>;
```

Comment:
Just as a note, we will need to do something more efficient later on

Reply:
Sure